src/seedu/addressbook/commands/ClearCommand.java

```diff
@@ -16,4 +16,9 @@ public CommandResult execute() {
         addressBook.clear();
         return new CommandResult(MESSAGE_SUCCESS);
     }
+
+    @Override
+    public boolean isMutating() {
+        return true;
```

Comment:
This method should have a header comment. Only then users of the overridden method will know how it exactly behaves.